### PR TITLE
Added new diagrams

### PR DESCRIPTION
**Pull request by renvrant**
Added a diagram to show selectors and another to show the full path

### CODE REVIEW

File: src/content/introduction.md [hunk 1]
```diff
@@ -40,8 +40,12 @@
 
 ---
 
+![Redux Flow](content/images/full-redux-flow.png "Redux Flow")
```
Comment (renvrant) [inline src/content/introduction.md:43]: I'm not sure this should go in the introduction. We can maybe even use this full circle diagram at the end, or in place of the other selector diagram I did...

File: src/content/introduction.md [hunk 2]
```diff
@@ -40,8 +40,12 @@
...
+
+---
+
 ## Not in this talk
-- Redux 101
+- Intro to Redux
```
Comment (renvrant) [inline src/content/introduction.md:48]: Should we try to briefly explain the concept using one of the diagrams? What do you think @danielfigueiredo ?

File: src/content/multi-entry.md [hunk 3]
```diff
@@ -56,12 +56,12 @@ case 'ADD_INDEXED_FORM_VALUE':
...
 
 ```ts
-CASE 'UPDATE_INDEXED_FORM_VALUE':
+case 'ADD_INDEXED_FORM_VALUE':
   const lensForProp = lensPath(action.payload.path);
   const propValue = <any[]> view(lensForProp, state);
   return assocPath(
     action.payload.path,
-    update(action.payload.index, action.payload.value, propValue),
+    concat(propValue, [action.payload.value]),
```
Comment (renvrant) [inline src/content/multi-entry.md:64]: I split this example into 2 slides, but I wonder if we can show even less

File: src/content/introduction.md (hunk 2 above)
Comment (danielfigueiredo) [reply src/content/introduction.md:48]: I like this idea, I think the diagram is pretty dope and would help us a lot explaining briefly the concepts of redux.

File: src/content/introduction.md (hunk 1 above)
Comment (danielfigueiredo) [reply src/content/introduction.md:43]: I wish all these diagrams were the wizard + dragon diagram with the selectors step included ❤️

File: src/content/multi-entry.md [hunk 4]
```diff
@@ -41,12 +41,12 @@ export const removeFromArray = (index, path) => ({
 ```ts
 import { concat, remove, update } from 'ramda';
 
-case 'ADD_INDEXED_FORM_VALUE':
+CASE 'UPDATE_INDEXED_FORM_VALUE':
```
Comment (danielfigueiredo) [inline src/content/multi-entry.md]: is this CASE intentionally uppercase?

File: src/content/multi-entry.md [hunk 5]
```diff
@@ -41,12 +41,12 @@ export const removeFromArray = (index, path) => ({
...
   const lensForProp = lensPath(action.payload.path);
   const propValue = <any[]> view(lensForProp, state);
   return assocPath(
     action.payload.path,
-    concat(propValue, [action.payload.value]),
+    update(action.payload.index, action.payload.value, propValue),
```
Comment (danielfigueiredo) [inline src/content/multi-entry.md:49]: nice!

**Review by danielfigueiredo — APPROVED** [review]
Thanks for updating this, just the switch case thing you might need to change but everything else seems good to me :)

File: src/content/multi-entry.md (hunk 4 above)
Comment (renvrant) [reply src/content/multi-entry.md]: Nope, not sure how that happened... Good catch

File: src/content/introduction.md (hunk 1 above)
Comment (renvrant) [reply src/content/introduction.md:43]: I want to put one right at the end!

**Pull request merged**